setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 setup(name='tap-salesforce',
-      version='2.0.1',
```

Comment:
This P.R qualify's for  a minor version release, since we are changing a lot of things.
I will suggest to use `2.1.0` over `2.0.1`

Reply:
Agree. I have updated it